src/test/resources/application-connectedtest.properties

```diff
@@ -2,6 +2,7 @@
 ct.ingestBucket=jade-testdata
 ct.ingestRequesterPaysBucket=jade_testbucket_requester_pays
 ct.googleBillingAccountId=00708C-45D19D-27AAFA
+ct.secondGoogleBillingAccountId=01A82E-CA8A14-367457
```

Comment:
This is the billing account that is only supposed to be used in billing account testing, right?

Maybe add some documentation here? I wouldn't want someone to naively spend a bunch of money against that account. Maybe we should call it the "noSpend" account or some such.

Reply:
Oh, I like that idea! I've rename the variable and added documentation.